lib/presets.ts

```diff
@@ -552,10 +552,6 @@ export function setFileNames(set: RegisterFileNameIcon) {
 		[IconSet.ColorLight]: '#0083e8',
 		[IconSet.IconDev]: 'devicons devicons-brackets'
 	});
-	set('.brackets', ['json'], {
```

Comment:
I'm not sure whether this can be removed, as `.brackets.json` and `brackets.json` are files used by brackets if I'm not mistaken. It'd be better to change the previous `set` call to:
```typescript
set(['.brackets', 'brackets'], ['json'], { ... });